pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -733,7 +733,8 @@ func TestChangefeedCursor(t *testing.T) {
 		// statement timestamp, so only verify this for enterprise.
 		if e, ok := fooLogical.(cdctest.EnterpriseTestFeed); ok {
 			var bytes []byte
-			sqlDB.QueryRow(t, `SELECT payload FROM system.jobs WHERE id=$1`, e.JobID()).Scan(&bytes)
+			sqlDB.QueryRow(t, `SELECT payload FROM "".crdb_internal.system_jobs WHERE id = $1`,
```

Comment:
curious: why do we need "" ?

Reply:
Added a comment to `jobs_verification.go` and changed most test places to use that query instead.